src/test/scala/it/pagopa/interop/tenantprocess/provider/TenantCreationSpec.scala

```diff
@@ -1026,7 +1027,7 @@ class TenantCreationSpec extends AnyWordSpecLike with SpecHelper with ScalatestR
     mockGetTenantByExternalId(PersistentExternalId(seed.externalId.origin, seed.externalId.value), tenant)
 
     Get() ~> tenantService.selfcareUpsertTenant(seed) ~> check {
-      assert(status == StatusCodes.OK)
```

Comment:
Am I missing something?
The behaviour has changed and looks good to me, but I was wondering why it was expected to be OK.